packages/wonder-blocks-tooltip/components/tooltip-anchor.test.js

```diff
@@ -0,0 +1,560 @@
+                            forceAnchorFocusivity={true}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </div>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                if (!ref) {
+                    return;
+                }
+
+                // Act
+                const result = ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBe("0");
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+
+        test("if tabindex already set, leaves it as-is", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <div tabIndex="-1">This is the anchor</div>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+                const nodes = (
+                    <div>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={true}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </div>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                if (!ref) {
+                    return;
+                }
+
+                // Act
+                const result = ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBe("-1");
+                done();
+            };
+
+            arrange(actAndAssert);
+        });
+    });
+
+    describe("forceAnchorFocusivity is false", () => {
+        test("does not set tabindex on anchor target", (done) => {
+            // Arrange
+            const arrange = (actAndAssert) => {
+                const fakeTooltipPortalMounter = (((
+                    <div>This is the anchor</div>
+                ): any): React.Element<typeof TooltipPortalMounter>);
+                const nodes = (
+                    <div>
+                        <TooltipAnchor
+                            forceAnchorFocusivity={false}
+                            anchorRef={actAndAssert}
+                        >
+                            {(active) => fakeTooltipPortalMounter}
+                        </TooltipAnchor>
+                    </div>
+                );
+                mount(nodes);
+            };
+
+            const actAndAssert = (ref) => {
+                if (!ref) {
+                    return;
+                }
+
+                // Act
+                const result = ref.getAttribute("tabindex");
+
+                // Assert
+                expect(result).toBeNull();
```

Comment:
This should probably be in the docs as I thought `forceAnchorFocusivity={false}` wouldn't actually do anything.

Reply:
I was expecting the docs stating "defaults to true" to show that it should do something. Why else would the prop exist?

Reply:
I've updated the docs to clarify these.

Reply:
I get what you're saying, but it wasn't obvious from the docs what should happen when the prop is `false`.